src/LegacyDeserializerFactory.php

```diff
@@ -41,7 +33,7 @@ public function __construct( Deserializer $dataValueDeserializer, EntityIdParser
 	}
 
 	/**
-	 * @return DispatchableDeserializer
+	 * @return Deserializer
 	 */
 	public function newEntityDeserializer() {
```

Comment:
@Benestar, @Jonas-WMDE, @JeroenDeDauw, merging this was a mistake. Users of these methods should not assume they get an actual DispatchableDeserializer. This should be an implementation detail.